src/styles/blocks/card.scss

```diff
@@ -0,0 +1,124 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 50px 0 0;
+
+  margin: 0 auto;
+
+  max-width: 944px;
```

Comment:
don't hardcode max-width, your catalog container should work fine for any number of cards